client/me/concierge/book/info-step.js

```diff
@@ -43,19 +45,26 @@ class InfoStep extends Component {
 	};
 
 	canSubmitForm = () => {
-		const { signupForm } = this.props;
-		if ( ! signupForm.message ) {
-			return false;
-		}
-		return !! signupForm.message.trim();
+		const { signupForm: { firstname, message } } = this.props;
+
+		return !! firstname.trim() && !! message.trim();
 	};
 
 	componentDidMount() {
+		const { userSettings } = this.props;
+
 		this.props.recordTracksEvent( 'calypso_concierge_book_info_step' );
+
+		// Prefill the firstname & lastname fields by user settings.
+		this.props.updateConciergeSignupForm( {
```

Comment:
I guess this is not a big deal but we could check if we don't already have the data in redux, otherwise if the customer goes back from the calendar step the typed first and last name will be overwritten by the ones in the user settings.

Reply:
Good idea. Updated in https://github.com/Automattic/wp-calypso/pull/22159/commits/2e3aba86cf06f8ae0abc886db21ef2ed41743942